ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -172,8 +174,12 @@ protected void onSelectionChanged(int selStart, int selEnd) {
   protected void onFocusChanged(
       boolean focused, int direction, Rect previouslyFocusedRect) {
     super.onFocusChanged(focused, direction, previouslyFocusedRect);
-    if (focused && mSelectionWatcher != null) {
-      mSelectionWatcher.onSelectionChanged(getSelectionStart(), getSelectionEnd());
+    if (focused) {
+      if (mClearTextOnFocus) {
+        setText("");
```

Comment:
Is this the right call?  This certainly will update the EditText, but will it properly update js?